scope.Rmd

```diff
@@ -1,3 +1,9 @@
+```{r include=FALSE, cache=FALSE}
+knitr::opts_chunk$set(
+  cache = TRUE,
+  echo = FALSE
+)
+```
```

Comment:
You don't need this because thanks to @maelle it's in [`_common.R`](https://github.com/ropenscilabs/statistical-software-review/blob/master/_common.R) and called via
https://github.com/ropenscilabs/statistical-software-review/blob/5bd6f8dfc6780d6b48b08bcf6495153ce0863caf/_bookdown.yml#L8

Reply:
learnt via https://community.rstudio.com/t/bookdown-caching/43652/2